PotreeConverterPartitioned/potreeconverterpartitioned.py

```diff
@@ -0,0 +1,533 @@
+                exit(1)
+            else:
+                self.partitionsCSV = config["PARTITIONS"]["CSV"]
+
+            self.countingBatchSize = eval(config["PARTITIONS"]["CountingBatchSize"])
+
+        if "TMP_STORAGE" not in sections:
+            LoggingWrapper.error("TMP_STORAGE section not found in config file")
+            exit(1)
+        else:
+            if "TmpDir" not in config["TMP_STORAGE"]:
+                LoggingWrapper.error("TmpDir not found in TMP_STORAGE section")
+                exit(1)
+            else:
+                self.tmpDir = config["TMP_STORAGE"]["TmpDir"]
+                if config["INPUT_OUTPUT"]["InputDirType"] == "local":
+                    self.tmpInputDir = self.InputDir
+                    self.inputToTmpInputSizeRatio = 0
+                else:
+                    self.tmpInputDir = self.tmpDir + "/PotreeConverter_Input"
+                    self.inputToTmpInputSizeRatio = 1
+                if config["INPUT_OUTPUT"]["OutputDirType"] == "local":
+                    self.tmpOutputDir = self.OutputDir
+                    self.inputToOutputSizeRatio = 0
+                else:
+                    self.tmpOutputDir = self.tmpDir + "/PotreeConverter_Output"
+                    self.inputToOutputSizeRatio = 2
+                self.lazHeadersDir = self.tmpDir + "/" + Path(self.InputDir).name + "_headers"
+            if "MaxTmpSpaceAvailable" not in config["TMP_STORAGE"]:
+                LoggingWrapper.error(
+                    "MaxTmpSpaceAvailable not found in TMP_STORAGE section")
+                exit(1)
+            else:
+                self.maxTmpSpaceAvailable = eval(config["TMP_STORAGE"]["MaxTmpSpaceAvailable"])
+            if "LazCompressionRatio" not in config["TMP_STORAGE"]:
+                LoggingWrapper.error(
+                    "LazCompressionRatio not found in TMP_STORAGE section")
+                exit(1)
+            else:
+                self.lazCompressionRatio = eval(config["TMP_STORAGE"]["LazCompressionRatio"])
+            if "DistributionBatchSize" not in config["TMP_STORAGE"]:
+                LoggingWrapper.error(
+                    "DistributionBatchSize not found in TMP_STORAGE section")
+                exit(1)
+            else:
+                self.distributionBatchSize = eval(config["TMP_STORAGE"]["DistributionBatchSize"])
+            if "CountingBatchSize" not in config["TMP_STORAGE"]:
+                LoggingWrapper.error(
+                    "CountingBatchSize not found in TMP_STORAGE section")
+                exit(1)
+            else:
+                self.countingBatchSize = eval(config["TMP_STORAGE"]["CountingBatchSize"])
+
+
+        if "COPIER" not in sections:
+            LoggingWrapper.error("COPIER section not found in config file")
+            exit(1)
+        else:
+            if "Type" not in config["COPIER"]:
+                LoggingWrapper.error("Type not found in COPIER section")
+                exit(1)
+            else:
```

Comment:
Combined with the copier methods this implies that processing requires all files. to be available on a shared filesystem. Given data volumes adding support for mass storage systems could be advisable (for future development)